traefik.go

```diff
@@ -114,13 +137,19 @@ Complete documentation is available at https://traefik.io`,
 
 	traefikConfiguration.ConfigFile = toml.ConfigFileUsed()
 
-	kv, err := CreateKvSource(traefikConfiguration)
+	kv, err = CreateKvSource(traefikConfiguration)
 	if err != nil {
 		fmtlog.Println(err)
 		os.Exit(-1)
 	}
 
-	if kv != nil {
+	usedCmd, err := f.GetCommand()
+	if err != nil {
+		fmtlog.Println(err)
+		os.Exit(-1)
+	}
+	// IF a KV Store is enable and no sub-command called in agrs
+	if kv != nil && usedCmd == traefikCmd {
 		s.AddSource(kv)
 		if _, err := s.LoadConfig(); err != nil {
 			fmtlog.Println(err)
```

Comment:
You'r right ;)
And same there https://github.com/containous/traefik/pull/551/files#diff-dba3f2b16983e1351d5b02a5700d18ecR134 I think !